MediaBrowser.MediaEncoding/Encoder/MediaEncoder.cs

```diff
@@ -357,15 +318,31 @@ private string GetProbePathFromEncoderPath(string appPath)
                 .FirstOrDefault(i => string.Equals(Path.GetFileNameWithoutExtension(i), "ffprobe", StringComparison.OrdinalIgnoreCase));
         }
 
-        private void LogPaths()
+        /// <summary>
+        /// Search the system $PATH environment variable looking for given filename.
+        /// </summary>
+        /// <param name="fileName"></param>
+        /// <returns></returns>
+        private string ExistsOnSystemPath(string fileName)
         {
-            _logger.LogInformation("FFMpeg: {0}", FFMpegPath ?? "not found");
-            _logger.LogInformation("FFProbe: {0}", FFProbePath ?? "not found");
+            var values = Environment.GetEnvironmentVariable("PATH");
+
+            foreach (var path in values.Split(Path.PathSeparator))
+            {
+                var candidatePath = GetEncoderPathFromDirectory(path);
```

Comment:
can you explain why it doesn't pass `fileName` inside? this seems to be searching for `ffmpeg` no matter what you asked it to do

Reply:
Originally I had lots of code in here to 'find files on path and loop through them. I finally replaced all that with call to existing GetEncoderPathFromDirectory() which you rightly say doesn't need a filename. Shall remove the function argument.

Reply:
Ended up keeping argument as moved to using ENUM to describe mpeg/probe and this function now gets called for both.